app/scripts/controllers/edit/buildConfig.js

```diff
@@ -522,6 +549,23 @@ angular.module('openshiftConsole')
       return sourceMap;
     };
 
+    $scope.addWebhookTrigger = function(webhookLabel) {
+      if (!webhookLabel) {
+        return;
+      }
+      var webhook = {
+        disabled: false,
+        data: {
+          type: webhookLabel
+        }
+      };
+      var webhookType = _.find($scope.createTriggerSelect.options, function(o) {return o.label === webhookLabel}).type
```

Comment:
Missing `;`. You can use the shorthand here

```js
var webhookType = _.find($scope.createTriggerSelect, { label: webhookLabel }).type;
...
```

or if you rename the properties as above, this would be

```js
var property = _.find($scope.createTriggerSelect, { type: type }).property;
```